aptos-move/aptos-release-builder/src/components/mod.rs

```diff
@@ -59,14 +79,54 @@ impl ReleaseConfig {
         }
         Ok(())
     }
+
+    pub fn load_config<P: AsRef<Path>>(path: P) -> Result<Self> {
+        // Open the file and read it into a string
+        let config_path_string = path.as_ref().to_str().unwrap().to_string();
+        let mut file = File::open(&path).map_err(|error| {
+            anyhow!(
+                "Failed to open config file: {:?}. Error: {:?}",
+                config_path_string,
+                error
+            )
+        })?;
+        let mut contents = String::new();
+        file.read_to_string(&mut contents).map_err(|error| {
+            anyhow!(
+                "Failed to read the config file into a string: {:?}. Error: {:?}",
+                config_path_string,
+                error
+            )
+        })?;
+
+        // Parse the file string
+        Self::parse(&contents)
+    }
+
+    pub fn save_config<P: AsRef<Path>>(&self, output_file: P) -> Result<()> {
+        let contents =
+            serde_yaml::to_vec(&self).map_err(|e| anyhow!("failed to generate config: {:?}", e))?;
+        let mut file = File::create(output_file.as_ref())
+            .map_err(|e| anyhow!("failed to create file: {:?}", e))?;
+        file.write_all(&contents)
+            .map_err(|e| anyhow!("failed to write file: {:?}", e))?;
+        Ok(())
+    }
+
+    pub fn parse(serialized: &str) -> Result<Self> {
+        serde_yaml::from_str(serialized).map_err(|e| anyhow!("Failed to parse the config: {:?}", e))
+    }
 }
 
 impl Default for ReleaseConfig {
     fn default() -> Self {
```

Comment:
Can you remind me where this is used again?

Reply:
This is used in the upgrade test to make sure that the proposal generation process would always work. Though in the future we should be replacing this with the real release config we want to put for the next release cycle.

Reply:
"in the future we should be replacing this with the real release config we want to put for the next release cycle." => What do you mean? Isn't this the default only?

Reply:
Sry I mean that right now, we use `ReleaseConfig::default` in the testing pipeline. We should expect to see another test that uses sth like `ReleaseConfig::upcoming_release` to make sure the config is indeed valid. Hope that makes sense.